ci/_environment.sh

```diff
@@ -5,5 +5,5 @@ if [[ -z "${DOCKER_REGISTRY+x}" ]]; then
     exit 1
 fi
 
-IMAGE_ANDROID_BUILDER=${DOCKER_REGISTRY}/android/builder:f1471cbeab
+IMAGE_ANDROID_BUILDER=${DOCKER_REGISTRY}/android/builder:6c5b5e3d02
```

Comment:
i didn't find any changes with an image? what is updated?

Reply:
removed setfacl?